index.rst

```diff
@@ -7,10 +7,14 @@ Rules of Engagement for Accessing Data During the Embargo Period
 
    Data must be embargoed after arriving at the USDF - for 30 days in commissioning and 80 hrs in Survey Operations. Alert Processing will not record vetoed streaks. What rules will we impose on accessing data during that period?
 
-Embargo Restrictions:
-=====================
+Embargo Restrictions
+====================
 
-Embargoed data types cannot leave the USDF during the embargo period. All Rubin USDF account holders will have full access to data in the embargo rack. Any data can reside transiently on batch or kubernetes processing nodes but must be deleted after use.
+Embargoed data types cannot leave the USDF during the embargo period.
+LSST Users, such as those who have been granted USDF accounts via the Resource Allocation Process, will not have access to the data in the embargo rack.
+Rubin Observatory staff (including the Commissioning Team) will have full access to the data in the embargo rack as their Rubin roles require. 
```

Comment:
"As their Rubin roles require" is a generalization inspired by previous comments from @leannep - no need to single out any particular Rubin roles here, it's just the case that not everyone on the Observatory staff will need access to the embargo rack. FAO @womullan and @richardxdubois

Reply:
Yes - this needs discussion (and may need a separate PR).